packages/shared/lib/core/wallet/stores/main-account-id.store.ts

```diff
@@ -0,0 +1,13 @@
+import { get, writable } from 'svelte/store'
+import { updateActiveWalletPersistedData } from '../../profile'
+import { selectedWalletId } from './selected-wallet-id.store'
+
+export const mainAccountId = writable<string | null>(null)
+
+export function updateMainAccountId(payload: string): void {
+    mainAccountId.set(payload)
+    // Update persisted data
+    updateActiveWalletPersistedData(get(selectedWalletId), {
```

Comment:
something weird is happening here I think
![Captura de pantalla de 2024-01-24 12-19-27](https://github.com/iotaledger/firefly/assets/100352899/9e224e81-483b-4ec7-81de-5f56e474a0e8)
This creates a wallet with null id